src/encoded/types/quality_metric.py

```diff
@@ -88,6 +133,10 @@ def _update(self, properties, sheets=None):
         # set name based on what is entered into title
         properties['overall_quality_status'] = overall
         super(QualityMetricBamcheck, self)._update(properties, sheets)
+    
+    @calculated_property(schema=QC_SUMMARY_SCHEMA)
+    def quality_metric_summary(self, request):
+        return
```

Comment:
Same here. Are you indicating in this that you expect this calculated property to be patched at a later time? What does this do on local deployment? I don't think this should have any effect but admit I haven't tried it. Will this just map the field in case we add it in the future?